### DISCUSSION TITLE
Gmail Verification Doesn't Appear

### DISCUSSION THREAD

**Discussion by YezGotIt** [anonaddy/anonaddy#665] [General]
Hi all,

I've been trying to get Gmail verified at this link: [https://app.addy.io/recipients](https://app.addy.io/recipients) for almost 30 minutes, but the verification email doesn't appear in Gmail. However, when I try with a 10-minute mail, it sends the verification email. I suspect that Gmail's provider doesn't support this. If anyone finds a solution, please post it here.

**Comment by YezGotIt** [comment]
1-Image:

![R1](https://github.com/user-attachments/assets/f7b8752b-d6af-44c2-aa1b-c7cfc58a06dc)

2-Image:

![R2](https://github.com/user-attachments/assets/799e67a6-ca9d-4879-9354-6a2c1d6615f2)

3-Image:

![R3](https://github.com/user-attachments/assets/bdbf6c4e-e2f4-4c69-a4a1-6c155961711a)

4-Image:

![R4](https://github.com/user-attachments/assets/8ed683da-8851-4637-9768-0bded0d00780)

**Comment by willbrowningme** [comment]
Have you checked you spam and any other folders in Gmail?

If it still isn't there please [send me an email](https://addy.io/contact/) with details of your Gmail address so I can check the logs.

**Reply by YezGotIt** [reply]
Yeah, it was in the spam folder. You are right. I'm sorry for the inconvenience. Thanks.
